app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -128,14 +129,11 @@ def self.verify_credentials(args)
   #
   def self.raw_connect(opts)
     # Create the connection:
-    connection = Connection.new(
+    Connection.new(
       :host  => opts[:server],
       :port  => opts[:port],
       :token => ManageIQ::Password.try_decrypt(opts[:token])
     )
-
-    # Verify that the connection works:
-    connection.valid?
```

Comment:
This was doing some of the validation but ended up not returning the `connection` object.  Rewritten to have raw_connect simply return the connection and moved all verification logic into `.verify_credentials` above